### ISSUE TITLE
CRS compatibility issue on rc2 branch

### ISSUE THREAD

**Issue by jptosso** [corazawaf/coraza#154]
FTW Output:
```
3 test(s) failed to run: ["953120-0" "953120-2" "953120-4"]
```

## Rule 953120
```
# Detect the presence of the PHP open tag "<? ", "<?= " or "<?php " in output.
#
# To prevent false positives (due to the short "<?" sequences), we also include,
# the space after it in an attempt to stop alerts in binary output.
#
#
SecRule RESPONSE_BODY "@rx <\?(?:=|php)?\s+" \
    "id:953120,\
    phase:4,\
    block,\
    capture,\
    t:none,\
    msg:'PHP source code leakage',\
    logdata:'Matched Data: %{TX.0} found within %{MATCHED_VAR_NAME}',\
    tag:'application-multi',\
    tag:'language-php',\
    tag:'platform-multi',\
    tag:'attack-disclosure',\
    tag:'paranoia-level/1',\
    tag:'OWASP_CRS',\
    tag:'capec/1000/118/116',\
    tag:'PCI/6.5.6',\
    ctl:auditLogParts=+E,\
    ver:'OWASP_CRS/3.4.0-dev',\
    severity:'ERROR',\
    setvar:'tx.outbound_anomaly_score_pl1=+%{tx.error_anomaly_score}',\
    setvar:'tx.anomaly_score_pl1=+%{tx.error_anomaly_score}'"
```

```yaml
  - test_title: 953120-0
    desc: "Just something that returns <?php "
    stages:
      - stage:
          input:
            dest_addr: "127.0.0.1"
            port: 80
            headers:
              Host: "localhost"
              User-Agent: "ModSecurity CRS 3 Tests"
              Accept: "text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5"
              Accept-Charset: "ISO-8859-1,utf-8;q=0.7,*;q=0.7"
              Accept-Encoding: "gzip,deflate"
              Accept-Language: "en-us,en;q=0.5"
              Content-Type: "text/plain"
            method: "GET"
            version: "HTTP/1.0"
            uri: "/anything"
            data: <?php echo "Hello World!\n" ?>
          output:
            log_contains: "id \"953120\""
```

**Test:**
```yaml
  - test_title: 953120-2
    desc: "Positive test, returns <? "
    stages:
      - stage:
          input:
            dest_addr: "127.0.0.1"
            port: 80
            headers:
              Host: "localhost"
              User-Agent: "ModSecurity CRS 3 Tests"
              Accept: "text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5"
              Accept-Charset: "ISO-8859-1,utf-8;q=0.7,*;q=0.7"
              Accept-Encoding: "gzip,deflate"
              Accept-Language: "en-us,en;q=0.5"
              Content-Type: "text/plain"
            method: "GET"
            version: "HTTP/1.0"
            uri: "/anything"
            data: "<? "
          output:
            log_contains: "id \"953120\""
```

```yaml
  - test_title: 953120-4
    desc: "Positive test, returns <?= "
    stages:
      - stage:
          input:
            dest_addr: "127.0.0.1"
            port: 80
            headers:
              Host: "localhost"
              User-Agent: "ModSecurity CRS 3 Tests"
              Accept: "text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5"
              Accept-Charset: "ISO-8859-1,utf-8;q=0.7,*;q=0.7"
              Accept-Encoding: "gzip,deflate"
              Accept-Language: "en-us,en;q=0.5"
              Content-Type: "text/plain"
            method: "GET"
            version: "HTTP/1.0"
            uri: "/anything"
            data: <?= "Hello World!\n" =?>
          output:
            log_contains: "id \"953120\""
```

**Comment by jptosso** [comment]
Added httpbin to the tests but the error is persisting

**Comment by jptosso** [comment]
Tests temporarily removed from ftw, they are not compatible with the test engine yet.

**Comment by fzipi** [comment]
This is still happening.

**Comment by jptosso** [comment]
I'm taking care of this one

**Comment by fzipi** [comment]
This was fixed, right?

**Comment by jptosso** [comment]
Merged into #218
